Bottom: 839ac427e619daeaadd55f47d3525b8e555a4b20
Top:    7098e8a662670ad67737099e6776a5a0dd6e004c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:16:57 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
